v0/destinations/google_adwords_offline_conversions/transform.js

```diff
@@ -0,0 +1,138 @@
+const { EventType } = require("../../../constants");
+const {
+  CustomError,
+  getHashFromArrayWithDuplicate,
+  constructPayload,
+  removeHyphens,
+  getValueFromMessage,
+  defaultRequestConfig,
+  getErrorRespEvents,
+  getSuccessRespEvents
+} = require("../../util");
+const {
+  trackClickConversionsMapping,
+  CLICK_CONVERSION,
+  trackCallConversionsMapping,
+  CALL_CONVERSION
+} = require("./config");
+const { validateDestinationConfig, getAccessToken } = require("./utils");
+
+const getConversions = (message, metadata, { Config }, conversionType) => {
+  let payload;
+  let endpoint;
+  const filteredCustomerId = removeHyphens(Config.customerId);
+  if (conversionType === "click") {
+    payload = constructPayload(message, trackClickConversionsMapping);
+    endpoint = CLICK_CONVERSION.replace(":customerId", filteredCustomerId);
+  } else {
+    payload = constructPayload(message, trackCallConversionsMapping);
+    endpoint = CALL_CONVERSION.replace(":customerId", filteredCustomerId);
+  }
+
+  payload.partialFailure = true;
+
+  const response = defaultRequestConfig();
+  response.endpoint = endpoint;
+  response.params = { event: message.event, customerId: filteredCustomerId };
```

Comment:
Do we need to send params?

Reply:
We need this if there is any network call via proxyRequest route. We need to change the implementation of transformerProxy to accept all types of keys apart from standard ones.